core/server/common/src/main/java/alluxio/cli/validation/HdfsValidationTask.java

```diff
@@ -50,13 +50,26 @@ public HdfsValidationTask() {}
 
   @Override
   public boolean validate(Map<String, String> optionsMap) {
+    if (shouldSkip()) {
+      return true;
+    }
     if (!validateHdfsSettingParity(optionsMap)) {
       System.err.format("Hdfs setting do not match.");
       return false;
     }
     return true;
   }
 
+  /**
+   * Checks whether this validation task should be skipped.
+   *
+   * @return true if underFS is NOT HDFS, false otherwise
+   */
+  protected boolean shouldSkip() {
+    String underFSURI = Configuration.get(PropertyKey.MASTER_MOUNT_TABLE_ROOT_UFS).toLowerCase();
```

Comment:
Use `AlluxioURI.getScheme()` to parse the ufs path.

Reply:
Done.